app/layout.tsx

```diff
@@ -1,23 +1,23 @@
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
-import "./globals.css";
-import React from "react";
+import type { Metadata } from "next"
+import { Inter } from "next/font/google"
+import "./globals.css"
+import React from "react"
 
-const inter = Inter({ subsets: ["latin"] });
+const inter = Inter({ subsets: ["latin"] })
 
 export const metadata: Metadata = {
   title: "CCV",
   description: "Center for Computation & Visualization",
-};
+}
 
 export default function RootLayout({
   children,
 }: Readonly<{
-  children: React.ReactNode;
+  children: React.ReactNode
```

Comment:
Probably not for this PR, but I like to type things like this as `React.propsWithChildren`. That's technically a type helper and will add the `children` type to whatever you pass into it (`React.propsWithChildren<MyComponentProps>`), but can be used just a a standalone type. Just my preference though!